test/sdf/triggered_camera_sensor_builtin.sdf

```diff
@@ -7,6 +7,7 @@
         <topic>/test/integration/TriggeredCameraPlugin_imagesWithBuiltinSDF</topic>
         <camera>
           <triggered>true</triggered>
+          <trigger_topic>/test/integration/TriggeredCameraPlugin_imagesWithBuiltinSDF/trigger</trigger_topic>
```

Comment:
I suppose this fixes it, but it's weird to me that the default trigger topic is `__default__/trigger`. I think this will need to be fixed in sdformat. I'll take a look at that.

Reply:
https://github.com/ignitionrobotics/sdformat/pull/1006